selenium/che-selenium-core/src/main/java/org/eclipse/che/selenium/core/client/TestWorkspaceServiceClient.java

```diff
@@ -257,6 +258,16 @@ public void ensureRunningStatus(Workspace workspace) throws IllegalStateExceptio
     }
   }
 
+  public static TestWorkspaceServiceClient create(
+      TestApiEndpointUrlProvider apiEndpointProvider,
```

Comment:
use container bindings instead of a static method.

Reply:
okay